crates/iota-protocol-config/src/lib.rs

```diff
@@ -2146,6 +2159,9 @@ impl ProtocolConfig {
                     // Enable normalization of PTB arguments in all networks.
                     cfg.feature_flags.normalize_ptb_arguments = true;
                 }
+                13 => {
```

Comment:
We don't need this part because the default value is 400 anyway. We will include it in this section once we need to change the value from the default. And we cannot change the default value.